test/test_unit.jl

```diff
@@ -875,7 +875,7 @@ isdir(outdir) && rm(outdir, recursive=true)
                           SVector(-1.2, 0.3)]
 
     for normal_direction in normal_directions
-      @test flux_hll(u, u, normal_direction, equations) ≈ flux(u, normal_direction, equations)
+      @test flux_hlle(u, u, normal_direction, equations) ≈ flux(u, normal_direction, equations)
```

Comment:
This was a mistake (by me) when I added this tests originally. Now we can actually run HLLE with normal directions supplied